### DISCUSSION TITLE
<Text slot"description"/> with a <Link/> causes tab focus to wrap on Select component and cause infinite loop of tab indexing.

### DISCUSSION THREAD

**Discussion by deltasierra96** [adobe/react-spectrum#5581] [Q&A] [answered]
When creating a Select component that has a description, when I add a <Link/> or even a standard anchor tag, the keyboards tab focus will indefinitely wrap inside the <Select/> component and not progress any further down the form.

The component structure is the same as in this example: [https://react-spectrum.adobe.com/react-aria/Select.html#description](url)

Is the best way to get around this issue to manually add a tabIndex={-1} to the <Link/> or should focus wrapping work another way?

Many thanks.

**Comment by snowystinger** [comment, marked as answer]
The description should not be an interactive element I think would be the recommendation. If there's that much information you need to communicate, help text probably isn't the way to go and you should instead have an always visible button with a popover with all of that information. See Spectrum's https://spectrum.adobe.com/page/contextual-help/ for an idea of how to handle this. You can include links inside of said popover as well.

**Reply by deltasierra96** [reply]
Perfect, @snowystinger The documentation is awesome for this library, but there's just so many patterns to learn and how to execute them. Thanks for your help!
